### DISCUSSION TITLE
CSV output on 1D problems

### DISCUSSION THREAD

**Discussion by dealmeidavf** [idaholab/moose#17036] [Q&A General] [answered]
Hello, 
I don't seem to get a CSV output (1D test problem).

```python
[Outputs]
  [csv]
   type = CSV
  []
[]
```
Thanks for inputs on this. I mean: can CSV be used with steady-state, 1D problems where nodal values are listed either in columns and node positions listed on rows (or vice-versa).

**Comment by dealmeidavf** [comment, marked as answer]
Since then I added this and it works:

```python
[VectorPostprocessors]
  [1d]
    type = LineValueSampler
    variable = 'u'
    start_point = '-5 0 0'
    end_point = '5 0 0'
    num_points = 11
    sort_by = id
  []
[]
```

Adapted from 

https://github.com/idaholab/moose/blob/130d29c48402b2ccf4cc1e71a21b23a212c36887/test/tests/outputs/csv_final_and_latest/final.i#L49-L60

**Reply by aeslaughter** [reply]
Thanks for posting the solution.
